klippy/kinematics/extruder.py

```diff
@@ -205,8 +209,16 @@ def move(self, print_time, move):
         self.extruder_move_fill(
             self.cmove, print_time, accel_t, cruise_t, decel_t, start_pos,
             start_v, cruise_v, accel, extra_accel_v, extra_decel_v)
-        self.stepper.step_itersolve(self.cmove)
+        for s in self.stepper_list:
+            s.step_itersolve(self.cmove)
         self.extrude_pos = start_pos + axis_d
+    def add_stepper(self, stepper):
+        if stepper == None:
+            return
+        stepper.set_max_jerk(9999999.9, 9999999.9)
+        stepper.setup_itersolve('extruder_stepper_alloc')
```

Comment:
I think these two lines should be issued in the ExtruderStepper __init__() method - as I'm not sure the order of operations would be correct if done from a connect() callback.

Reply:
Agreed, certainly more appropriate place, will fix that.